mautic/mautic
# Use ajax to populate email counts to prevent timeouts for loading the page

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? |  y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

If there are a lot of counts (pending, sent, read, etc) for an email and a lot of emails per page, the page may timeout waiting to load all the counts. This PR changes it to load the page then use ajax to update the counts one email at a time to prevent this time out.

[//]: # ( As applicable: )
#### Steps to test this PR:
1. View the emails page
2. Watch the counts update and they should be appropriate for the email
